Makefile

```diff
@@ -8,20 +10,28 @@ SPEC_PATH    := spec/
 
 .PHONY: default
 default: ci
-.PHONY: bash
-bash:
-	@$(COMPOSE_DEV) $(BASH)
 
 .PHONY: ci
 ci:
 	@$(BASH_TEST) "bin/rails db:setup db:migrate ci"
 
-.PHONY: clean
-clean:
-	rm -r data || true
-	$(COMPOSE_TEST) run vets-api rm -r coverage log tmp .git || true
+.PHONEY: ci-down
+ci-down:
 	$(COMPOSE_TEST) down
 
+.PHONY: ci-clean
+ci-clean:
+	$(COMPOSE_TEST) run vets-api git clean -fdx || true
+	$(COMPOSE_TEST) down
+
+.PHONY: ci-spec
+ci-spec:
+	@$(BASH_TEST) "bin/rspec ${SPEC_PATH}"
+
+.PHONY: bash
+bash:
+	@$(COMPOSE_DEV) $(BASH)
+
```

Comment:
Was moving the `bash` target intentional? Make targets are listed alphabetically (for better or worse).

Reply:
I moved things around to be grouped in "ci-" and "not ci-" fashion because that's how I tend to think.  I can re-ascibeticalize if you it works better for folks.

Reply:
I agree with you, but I'm mostly indifferent TBH. I'm good either way.